app/controllers/api/v1/mutes_controller.rb

```diff
@@ -8,10 +8,15 @@ class Api::V1::MutesController < Api::BaseController
   respond_to :json
 
   def index
-    @accounts = load_accounts
+    @data = @accounts = load_accounts
     render json: @accounts, each_serializer: REST::AccountSerializer
   end
 
+  def details
```

Comment:
How about using `Api::V2::MutesController` instead? e.g. `GET /api/v2/mutes`?

Reply:
Ah, good idea. I moved it to there in 2d003baddafbad335dc2cbf73c3d8db813d62229 and d24bfaae7f15f7548f949b6abb48c3c31d2c0a86